src/vm/core.zig

```diff
@@ -390,6 +399,26 @@ pub const CairoVM = struct {
     pub fn getPc(self: *const Self) Relocatable {
         return self.run_context.pc.*;
     }
+
+    pub fn deduce_memory_cell(
+        self: *Self,
+        address: relocatable.Relocatable,
+    ) CairoVMError!?relocatable.MaybeRelocatable {
+        for (self.builtin_runners.items) |builtin| {
+            if (@as(
+                u64,
+                builtin.base(),
+            ) == address.segment_index) {
+                return bitwise.deduce(
```

Comment:
Yes I understand the logic, I just changed the naming relating to that. So the import is a little strange (`const builtin = @import("./builtins/bitwise/bitwise.zig");`) because we import the bitwise file and give it the name `builtin` but that's fine correct itself when we have other implementations in the same folder.